pages/data-providers/[data-provider-id]/overview.jsx

```diff
@@ -1,17 +1,128 @@
 import React from 'react'
+import { ResponsiveContainer, RadialBarChart, RadialBar } from 'recharts'
+import { classNames } from '@oacore/design/lib/utils'
 
-import { Card } from 'design'
-
-const Dashboard = props => (
-  <Card {...props}>
-    <h1>CORE</h1>
-    <p>
-      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Mollitia numquam
-      aliquid, blanditiis praesentium doloribus molestiae sequi facere quasi
-      minima odio officia maxime accusantium iusto ducimus labore ut nemo
-      perferendis libero.
-    </p>
+import styles from './overview.css'
+
+import { Button, Card, Overlay, Numeral } from 'design'
+
+const Circle = ({ children }) => (
+  <div className={styles.circle}>
+    <div className={styles.inner}>{children}</div>
+  </div>
+)
+
+const Num = ({ value, append, caption, diff, ...restProps }) => (
+  <Numeral tag="p" {...restProps}>
+    <Numeral.Value>{value}</Numeral.Value>
+    {append && <Numeral.Appendix>&nbsp;{append}</Numeral.Appendix>}{' '}
+    {diff && <Numeral.Diff>{diff}</Numeral.Diff>}{' '}
+    <Numeral.Caption>{caption}</Numeral.Caption>
+  </Numeral>
+)
+
+const dataProviderStatistics = {
+  totalMetaDataCount: 5714,
+  totalFullTextCount: 2114,
+  diffMetaDataCount: 124,
+  diffFullTextCount: 14,
+}
+
+const PlaceholderCard = ({ title, value, description }) => (
+  <Card>
+    <h2>{title}</h2>
+    <Circle>
+      <Num value={value} append="%" caption={title} />
+    </Circle>
+    <p>{description}</p>
+    <Button variant="contained" disabled>
+      Browse
+    </Button>
+    <Overlay>
+      <b>Coming soon</b>
+    </Overlay>
+  </Card>
+)
+
+const DataStatisticsCard = ({
+  metaDataCount,
+  fullTextCount,
+  className,
+  ...restProps
+}) => (
+  <Card className={classNames.use(styles.dataCard, className)} {...restProps}>
+    <h2>Content at glance</h2>
+    <Num value={fullTextCount} caption="full texts" />
+    <Num value={metaDataCount} caption="meta-data records" size="small" />
+    <Button variant="contained" href="content" tag="a">
+      Browse
+    </Button>
   </Card>
 )
 
+const DepositingCard = ({ title = 'Depositing', description, value }) => {
+  const data = [
+    {
+      uv: 100,
+      fill: '#ffffffff',
+    },
+    {
+      name: '18-24',
+      uv: value,
+      fill: 'var(--primary)',
+    },
+  ]
+  return (
+    <Card>
+      <h2>{title}</h2>
+      <ResponsiveContainer width="100%" height={300}>
+        <RadialBarChart
+          innerRadius="10%"
+          outerRadius="80%"
+          data={data}
+          startAngle={225}
+          endAngle={-45}
+        >
+          <RadialBar minAngle={15} background clockWise dataKey="uv" />
+        </RadialBarChart>
+      </ResponsiveContainer>
```

Comment:
@Joozty `RadialBarChart` does not work as well as I expected. Unfortunately I have to fall back to own component.

Reply:
I would not give it up just like. Are you sure about this? What about other solutions like Piechart?

```
const DepositingCard = ({ title = 'Depositing', description, value }) => {
  const chartData = [
    { name: 'score', value: 680, fill: 'var(--primary)' },
    { name: 'max', value: 400, fill: 'var(--gray-100)' },
  ]
  return (
    <Card>
      <h2>{title}</h2>
      <ResponsiveContainer width="100%" height={300}>
        <PieChart margin={{ top: -25, right: -25, left: -25, bottom: 0 }}>
          <Pie
            data={chartData}
            cx="50%"
            cy="50%"
            startAngle={-135}
            endAngle={-405}
            innerRadius="60%"
            outerRadius="80%"
            stroke="none"
          />
        </PieChart>
      </ResponsiveContainer>
      <Num value={value} append="%" caption="papers" />
      <p>{description}</p>
      <Button variant="contained" href="deposit-dates" tag="a">
        Browse
      </Button>
    </Card>
  )
}
```

I did not style it properly. If you think it easier to use something else for you then go for it but I still think you should give it another try.